test/integration/build/aws.tf

```diff
@@ -1895,3 +1889,13 @@ resource "aws_ssm_document" "ssm_document_1" {
   }
 DOC
 }
+
+resource "aws_vpc_endpoint" "vpc_endpoint_1" {
+  count           = var.aws_enable_creation
+  vpc_id          = aws_vpc.inspec_vpc[0].id
+  service_name    = "com.amazonaws.${var.aws_region}.s3"
+  route_table_ids = [aws_route_table.route_table_first[0].id]
+  tags = {
+    Name = var.aws_vpc_endpoint_name
+  }
+}
```

Comment:
Probably a good idea to leave this to be added by the vpc_endpoint PR. Just to have it all added in a single PR.